<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>compreheNGSive</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="featureList">
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Feature Files:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add...</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_4" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="schemaView">
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>3</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>FILTER</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>Force Categorical</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Include Missing</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Include categorical:</string>
          </property>
          <property name="text">
           <string>PASS</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Include categorical:</string>
          </property>
          <property name="text">
           <string>HRun</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Include numeric:</string>
          </property>
          <property name="text">
           <string>&lt;all&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>QUAL</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>Force Categorical</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Include Missing</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Include categorical:</string>
          </property>
          <property name="text">
           <string>&lt;all&gt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Include numeric:</string>
          </property>
          <property name="text">
           <string>30.0</string>
          </property>
          <property name="text">
           <string>Inf</string>
          </property>
         </item>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>This view is currently disabled: for now, initial soft filters will be the top 5% of the starting X and Y axes, and all numeric values will be treated as such</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Schema:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Load</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget_8" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Variant File:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pathBox">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton">
           <property name="text">
            <string>Browse...</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Starting Y axis</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="yAxisChooser"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Starting X axis</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="xAxisChooser"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QWidget" name="widget_9" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>Quit</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
